test-utils/src/lib.rs

```diff
@@ -106,3 +106,13 @@ pub fn into_record_batch(engine_data: Box<dyn EngineData>) -> RecordBatch {
         .unwrap()
         .into()
 }
+
+/// We implement abs_diff here so we don't have to bump our msrv.
+/// TODO: Remove and use std version when msrv >= 1.81.0
```

Comment:
Do we have a tracking issue for this so we don't forget?
(really it would be neat if rust had a way to annotate these sorts of workarounds, so they start failing once MSRV bumps)

Reply:
https://github.com/delta-io/delta-kernel-rs/issues/598

Reply:
good call :)